packages/conf/src/index.ts

```diff
@@ -241,22 +276,218 @@ export function replaceRootDir(
 }
 
 /**
- * Replace fallbackLanguage with fallbackLocale
- *
- * Released in lingui-conf 0.9
- * Remove anytime after 3.x
+ * Replace fallbackLanguage and fallbackLocale, by the new standard fallbackLocales
+ * - https://github.com/lingui/js-lingui/issues/791
+ * - Remove anytime after 3.x
  */
-type DeprecatedFallbackLanguage = { fallbackLanguage: string | null }
+type DeprecatedFallbackLanguage = { fallbackLanguage?: string, fallbackLocale?: string }
 
 export function fallbackLanguageMigration(
   config: LinguiConfig & DeprecatedFallbackLanguage
 ): LinguiConfig {
-  const { fallbackLocale, fallbackLanguage, ...newConfig } = config
+  const { fallbackLocale, fallbackLanguage, fallbackLocales } = config
 
-  return {
-    ...newConfig,
-    fallbackLocale: fallbackLocale || fallbackLanguage || "",
+  if (fallbackLocales === false) return {
+    ...config,
+    fallbackLocales: null,
+  }
+
+  config.locales.forEach((locale) => {
+    const fl = getCldrParentLocale(locale.toLowerCase())
+    if (fl) {
+      config.fallbackLocales = {
+        ...config.fallbackLocales,
+        [locale]: fl
+      }
+    }
+  })
+
+  const DEFAULT_FALLBACK = fallbackLocales?.default || fallbackLocale || fallbackLanguage
+  if (DEFAULT_FALLBACK) {
+    if (!config.fallbackLocales) config.fallbackLocales = {}
+    config.fallbackLocales.default = DEFAULT_FALLBACK
   }
+
+  return config
+}
```

Comment:
I would remove `fallbackLanguage` completely. This change was introduced in Lingui 0.9 and there were two major bumbs since then. I don't think we need to keep it that long, what do you think?

If you agree, let's simplify it to converting `fallbackLocale` to `fallbackLocales` and just add one check if both `fallbackLocale` and `fallbackLocales.default` are set. That's the only case which should end up with hard error.